ui/src/ducks/app/nodes.js

```diff
@@ -252,6 +252,17 @@ export function* fetchNodes() {
             node.status.conditions.find(
               (conditon) => conditon.type === 'Ready',
             );
+
+          // Store the name of conditions which the status are True in the array
+          // given the avaiable conditions (Ready, DiskPressure, MemoryPressure, PIDPressure, Network Unavailable, Unschedulable)
```

Comment:
Oh no that's not what I'm saying. I'm just suggesting that we could drop the `Ready` condition from this `conditions` array, because we don't look at it (since we have what we need in the `status` prop already) :slightly_smiling_face: